packages/cli/src/tools/installPods.js

```diff
@@ -26,7 +26,9 @@ async function installPods({
     try {
       await commandExists('pod');
     } catch (e) {
-      loader.stop();
+      if (loader) {
+        loader.stop();
```

Comment:
A bit ugly, but I guess there's nothing we can do about it!

Reply:
Thought the same.

Reply:
you could add "onInstallFailure" and "onInstallSuccess" callbacks and delegate to them to make it cleaner, but I don't care about it here. It's not _that_ ugly provided it's a small helper used in 2 places, not a public API surface :)